src/graphql/models/User.js

```diff
@@ -158,6 +159,63 @@ const User = new GraphQLObjectType({
         'If not null, the user is blocked with the announcement in this string.',
       type: GraphQLString,
     },
+
+    badges: {
+      type: new GraphQLNonNull(new GraphQLList(new GraphQLNonNull(Badge))),
```

Comment:
According to [ES schema](https://github.com/cofacts/rumors-db/blob/master/schema/users.ts#L51-L55) and the GraphQL models defined in this PR, `badges` field under `users` should have `UserAwardedBadge` type.

```suggestion
      type: new GraphQLNonNull(new GraphQLList(new GraphQLNonNull(UserAwardedBadge))),
```

It is recommended that we add a unit test (and relevant fixtures) under https://github.com/cofacts/rumors-api/blob/master/src/graphql/queries/__tests__/GetUser.js to ensure that these newly added fields can be loaded as expected.